common/djangoapps/third_party_auth/tests/specs/test_testshib.py

```diff
@@ -111,7 +111,7 @@ def test_register(self):
         continue_response = self.client.get(TPA_TESTSHIB_COMPLETE_URL)
         # And we should be redirected to the dashboard:
         self.assertEqual(continue_response.status_code, 302)
-        self.assertEqual(continue_response['Location'], self.url_prefix + reverse('dashboard'))
+        self.assertEqual(continue_response['Location'], self.url_prefix + self.dashboard_page_url)
```

Comment:
You decided to leave this change in, right? That's fine, but it will mean this fork has a diff with upstream in more places.